tests/whitebox/integration/logical/test_create.py

```diff
@@ -132,12 +132,78 @@ def setUp(self):
         command_line = self._MENU + [self._POOLNAME] + self._VOLNAMES[0:1]
         RUNNER(command_line)
 
-    def testDestroy(self):
+    def testCreate(self):
         """
-        Creation of multiple volumes, of which one already exists, must fail.
+        Creation of 2 volumes, of which 1 already exists, must fail.
+        There is 1 target resource that would change.
+        There is 1 target resource that would not change.
         """
-        command_line = self._MENU + [self._POOLNAME] + self._VOLNAMES
+        command_line = self._MENU + [self._POOLNAME] + self._VOLNAMES[0:2]
+        with self.assertRaises(StratisCliActionError) as context:
+            RUNNER(command_line)
+        cause = context.exception.__cause__
+        self.assertIsInstance(cause, StratisCliPartialChangeError)
+        self.assertNotEqual(str(cause), "")
+
+    def test2Create(self):
+        """
+        Creation of 3 volumes, of which 1 already exists, must fail.
+        There are multiple (2) target resources that would change.
+        There is 1 target resource that would not change.
+        """
+        command_line = self._MENU + [self._POOLNAME] + self._VOLNAMES[0:3]
+        with self.assertRaises(StratisCliActionError) as context:
+            RUNNER(command_line)
+        cause = context.exception.__cause__
+        self.assertIsInstance(cause, StratisCliPartialChangeError)
+        self.assertNotEqual(str(cause), "")
+
+
+class Create5TestCase(SimTestCase):
+    """
+    Test creating a set of volumes, of which two already exist.
+    """
+
+    _MENU = ["--propagate", "filesystem", "create"]
+    _POOLNAME = "deadpool"
+    _VOLNAMES = ["oubliette", "mnemosyne", "gaia", "zeus"]
+
+    def setUp(self):
+        """
+        Start the stratisd daemon with the simulator.
+        """
+        super().setUp()
+        command_line = ["pool", "create", self._POOLNAME] + _DEVICE_STRATEGY()
+        RUNNER(command_line)
+
+        command_line = self._MENU + [self._POOLNAME] + self._VOLNAMES[0:1]
```

Comment:
Line 179 and 182 together need a comment to explain why you're calling filesystem create method twice. Otherwise, future readers will be very confused. And, should stratisd allow reation of more than one, then the comment would mark where we could get rid of this odd-looking construction. So it's a good idea  to leave some breadcrumbs for future maintainers here.